### ISSUE TITLE
Is the Guard Rules Registry dead?

### ISSUE THREAD

**Issue by hakandilek** [aws-cloudformation/aws-guard-rules-registry#279]
### link to reference doc page

README.md

### Describe your issue?

I could not find any hint on the README file or nowhere between the issues but the last commit seems to be months old and the latest release is more than 2 years old. The issues are also stalling for about a year without any comments/reactions on them.

Is AWS the Guard Rules Registry dead or living somewhere else? If the latter the case, it'd be nice to mention this somewhere in the documentation.

**Comment by zechariahkasina** [comment]
@hakandilek I don't think this registry is dead. Perhaps, this repo lost the traction from the owners. These rules are written on best effort basis as per the cfn-guard tool [documentation](https://github.com/aws-cloudformation/cloudformation-guard/tree/main?tab=readme-ov-file#registry). Also, this can a good opportunity to contribute to this repo.

**Comment by johanmynhardt** [comment]
What worries me is that there are 16 `.guard` files without any rules (up to [7f7340c26](https://github.com/aws-cloudformation/aws-guard-rules-registry/commit/7f7340c26ae5d5e8874651dbffeb12e0e9f505b6) on `main`). Without inspecting one just assumes there are rules (moral of the story: never assume, always confirm).

```shell
# fish shell
╰─λ set sep ';'
    for gf in (find . -name '*.guard')
        set rule_count (grep ^rule $gf | wc -l) || 0
        set rules (grep ^rule $gf | awk '{print $2}' | string join ', ')
        if test -z "$rules"
            set rules "---"
        end
        echo $gf$sep$rule_count$sep$rules
    end | sort | sort -t $sep -nk2 | awk -F $sep '{if ( $2 == 0 ) print $1 }'
```

Inspecting by hand it rings true that there are contents, but everything as comments. One can only speculate on what happened to the project 😬

But yes, just be aware.
